packages/Badge/src/Badge.spec.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { render, cleanup } from 'react-testing-library';
+import { render, cleanup } from '@testing-library/react';
```

Comment:
Need to back these changes out when the other PR goes through.

Reply:
Will do!